src/deadline/keyshot_submitter/Submit to AWS Deadline Cloud.py

```diff
@@ -0,0 +1,386 @@
+# AUTHOR AWS
+# VERSION 0.1.3
+# Submit to AWS Deadline Cloud
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+from dataclasses import dataclass
+import json
+import os
+from pathlib import Path
+import platform
+import shlex
+import subprocess
+import tempfile
+from typing import Any, Optional
+
+import lux
+
+RENDER_SUBMITTER_SETTINGS_FILE_EXT = ".deadline_render_settings.json"
+
+
+@dataclass
+class Settings:
+    scene_file: str
+    frames: str
+    output_file_path: str
+    output_format: str
+    input_filenames: list[str]
```

Comment:
Normally any of the JA lists are stored as sets, is there a reason I'm missing why these need to be lists?